app/src/main/java/com/github/amusinamaria/ui/MainActivity.kt

```diff
@@ -21,12 +27,22 @@ class MainActivity : AppCompatActivity(), MoviesListFragment.MovieClickListener
         }
     }
 
-    override fun onClickShowMovieDetails() {
+    fun showMovieDetails(movieCard: MovieCard) {
+        val detailsFragment = DetailsFragment()
+        val args = Bundle()
+        args.putParcelable(MOVIE_ARGS_KEY, movieCard)
+        detailsFragment.arguments = args
```

Comment:
Use DetailsFragment.newInstance(movieCard) and put newInstance fun to DetailsFragment companion object

Reply:
Done